src/client/camera.cpp

```diff
@@ -45,6 +45,140 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #define WIELDMESH_AMPLITUDE_X 7.0f
 #define WIELDMESH_AMPLITUDE_Y 10.0f
 
+// Returns the fractional part of x
+inline f32 my_modf(f32 x)
+{
+	double dummy;
+	return modf(x, &dummy);
+}
+
+WieldNode::WieldNode(HandIndex index, Client *client, scene::ISceneManager *mgr, video::SColor player_light_color) :
```

Comment:
player_light_color is probably not necessary here because it's updated in the first call to `update`

Reply:
I removed it